fees/jito/index.ts

```
@@ -0,0 +1,43 @@
+import { FetchOptions, SimpleAdapter } from "../../adapters/types"
+import { CHAIN } from "../../helpers/chains"
+import { getSqlFromFile, queryDuneSql } from "../../helpers/dune"
+
+const fetchRevenue = async (options: FetchOptions) => {
+  // JitoSOL revenue
+
+  // https://dune.com/queries/4908703
+  // https://dune.com/queries/4908750
+  const sql = getSqlFromFile("helpers/queries/jito.sql", {
+    start: options.startTimestamp,
+    end: options.endTimestamp
+  });
+
+  const data: any[] = (await queryDuneSql(options, sql));
+  if (data[0] == undefined) {
+    return {
+      dailyProtocolRevenue: 0,
```

same required changes as jito-restaking, we need dailyFees, and let it fail, instead of handling it. and update methodology too